test/tools/yulInterpreter/EwasmBuiltinInterpreter.cpp

```diff
@@ -494,18 +492,16 @@ u256 EwasmBuiltinInterpreter::evalEthBuiltin(string const& _fun, vector<uint64_t
 	return 0;
 }
 
-bool EwasmBuiltinInterpreter::accessMemory(u256 const& _offset, u256 const& _size)
+void EwasmBuiltinInterpreter::accessMemory(u256 const& _offset, u256 const& _size)
 {
 	if (((_offset + _size) >= _offset) && ((_offset + _size + 0x1f) >= (_offset + _size)))
```

Comment:
Ah this should check for exceeding a page size (64k).

Since this interpreter doesn't even support increasing the memory, that should be enough.

Reply:
Done.